repose-aggregator/components/filters/valkyrie-authorization/src/main/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilter.scala

```diff
@@ -75,7 +76,15 @@ class ValkyrieAuthorizationFilter @Inject()(configurationService: ConfigurationS
     val tracingHeader = nullOrWhitespace(Option(mutableHttpRequest.getHeader(CommonHttpHeader.TRACE_GUID.toString)))
     val urlPath: String = new URL(mutableHttpRequest.getRequestURL.toString).getPath
     val matchingResources: Seq[Resource] = Option(configuration.getCollectionResources)
-      .map(_.getResource.asScala.filter(_.getPathRegex.r.pattern.matcher(urlPath).matches())).getOrElse(Seq.empty[Resource])
+      .map(_.getResource.asScala.filter(resource => {
+        val pathRegex = resource.getPathRegex
+        pathRegex.getValue.r.pattern.matcher(urlPath).matches() && {
+          val httpMethods = pathRegex.getHttpMethods
+          httpMethods.isEmpty ||
```

Comment:
because if they have set it to empty, that would imply that they want it to happen for no http methods. I'm not sure how you were getting empty in system tests, that would make sense in unit tests though where a config isn't getting marshaled.